melodies_monet/util/tools.py

```diff
@@ -498,13 +498,90 @@ def convert_std_to_amb_bc(ds,convert_vars=[],temp_var=None,pres_var=None):
     #So I just need to convert the obs from std to amb.
     Losch = 2.69e25 # loschmidt's number
     #1013 mb, 273 K (101300 Pa)
-    std_bc = 101300.*6.02214e23/(8.314472*273.)
+    std_bc = 101300.*N_A/(R*273.)
     #use pressure_obs now, which is in pa
-    Airnum = ds[pres_var]*6.02214e23/(8.314472*ds[temp_var])
+    Airnum = ds[pres_var]*N_A/(R*ds[temp_var])
     
     # amb to std = Losch / Airnum
     convert_std_to_amb_bc = Airnum/std_bc
     
     for var in convert_vars:
         ds[var] = ds[var]*convert_std_to_amb_bc
 
+
+def calc_partialcolumn(modobj, var="NO2"):
+    """Calculates the partial column of a species from its concentration.
```

Comment:
Perhaps clarify that this is only the part of the column within each grid cell?  I've been working with literature where the partial column refers to the integral from the top of the atmosphere down to that point.  Either a new paragraph before the Parameters section, or an additional line in the Returns section would work